login/service.go

```diff
@@ -729,7 +744,78 @@ func (keycloak *KeycloakOAuthProvider) CreateOrUpdateKeycloakUser(accessToken st
 			}
 		}
 	}
-	return identity, user, nil
+	err = keycloak.notifyWITService(ctx, user, identity, WITEndpointUserProfile, &accessToken)
+	return identity, user, err
+}
+
+func (keycloak *KeycloakOAuthProvider) notifyWITService(ctx context.Context, user *account.User, identity *account.Identity, WITEndpointUserProfile string, accessToken *string) error {
```

Comment:
Let's create one function to notify WIT, something like UpdateUserInWIT(payload) and re-use it here and in user service instead of having two independent and overlapping NotifyWITService() methods.

Reply:
Yeah, that was the intention.. but maintained a bit of caution intially.
For user update api delegation : only specific fields were to be patched.
For login related user updates : we are updating almost the whole thing.

As suggested , brought it all into common code here : 
https://github.com/sbose78/fabric8-auth/blob/0fab9e3c83cde95f97cc099c6aa2366128136cd2/remoteservice/wit.go#L46